build_test.go

```diff
@@ -118,6 +127,17 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 					Metadata: map[string]interface{}{
 						"dependency-sha": "some-sha",
 					},
+					SBOM: sbom.Formatter{},
```

Comment:
I'd love to see one test here where the `SBOMFormats` in the build context are set, and then we can see that the build result layer has some context like we do here: https://github.com/paketo-buildpacks/node-engine/blob/71ab3af64a0dd37947f4329381209723878c7773/build_test.go#L141-L150